build.sbt

```diff
@@ -38,23 +38,41 @@ lazy val commonSettings = Seq(
 
 lazy val libats = (project in file("libats"))
   .enablePlugins(BuildInfoPlugin, Versioning.Plugin)
-  .configs(commonConfigs:_*)
+  .configs(commonConfigs: _*)
   .settings(commonSettings)
   .settings(Publish.settings)
 
 lazy val libats_messaging = (project in file("libats-messaging"))
   .enablePlugins(BuildInfoPlugin, Versioning.Plugin)
-  .configs(commonConfigs:_*)
+  .configs(commonConfigs: _*)
   .settings(commonSettings)
   .settings(Publish.settings)
 
 lazy val libats_messaging_datatype = (project in file("libats-messaging-datatype"))
   .enablePlugins(BuildInfoPlugin, Versioning.Plugin)
-  .configs(commonConfigs:_*)
+  .configs(commonConfigs: _*)
   .settings(commonSettings)
   .settings(Publish.settings)
   .dependsOn(libats_messaging)
 
+lazy val libats_metrics = (project in file("libats-metrics"))
+  .enablePlugins(BuildInfoPlugin, Versioning.Plugin)
+  .configs(commonConfigs: _*)
+  .settings(commonSettings)
+  .settings(
+    libraryDependencies ++=
+      Seq("metrics-core", "metrics-jvm").map("io.dropwizard.metrics" % _ % "3.2.2")
```

Comment:
this can be added in the `build.sbt` of the module, to make it consistent with what are doing with the other modules.